### DISCUSSION TITLE
Is there a usable demo of the mesh network?

### DISCUSSION THREAD

**Discussion by unlikely-alice** [tcfev/forDem#167] [Q&A]
Hey guys. I think this is a really cool project which could be immensely useful to Iranians right now. Is there a small demo or a proof of concept MVP of the mesh network I can mess around with? If not, do you have a timeline for when it will become usable?

I tried to use app.fordem.org, but it gets stuck at loading after the 12-word test thing.

**Comment by armantorkzaban** [comment]
We are working hard on one. We will commit something within 2-3 weeks.
